nethttp/client_test.go

```diff
@@ -92,10 +95,26 @@ func TestClientTrace(t *testing.T) {
 				if v != "GetConn" {
 					t.Fatalf("got %s expected, %s", v, "GetConn")
 				}
+
+				for k, expected := range tt.additionalTags {
+					result := span.Tag(k)
+					if expected != result {
+						t.Fatalf("got %v expected, %v, for key %s", result, expected, k)
```

Comment:
misplaced comma in `"expected, "`, probably should be `"got %v, expected %v, for key %s"`

looks like the existing checks also have the same typo

Reply:
:+1: fixed all the other ones as well.